composer.json

```diff
@@ -24,10 +24,10 @@
 		"irc": "irc://irc.freenode.net/wikidata"
 	},
 	"require": {
```

Comment:
I imagine some of these old versions of the other libraries probably want dropping at some point